cmd/root.go

```diff
@@ -88,7 +90,8 @@ func FormatterCmds() []*cobra.Command {
 
 func doPrint(path string, fn func(*tfconf.Module) (string, error)) {
 	options := &tfconf.Options{
-		Path: path,
+		Path:         path,
+		OutputValues: settings.OutputValues,
```

Comment:
You actually defined `OutputValuesPath string` in `tfconf.Options` but don't pass it from here. I'd suggest to create a corresponding flag for it and pass it down.

Reply:
Fixed this~